ramls/eholdings.raml

```diff
@@ -11,63 +11,10 @@ documentation:
 
 types:
   configuration: !include types/configuration/configuration.json
-  configurationPutRequest: !include types/configuration/configurationPutRequest.json
   jsonapiError: !include types/jsonapiError.json
```

Comment:
`configuration` and `jsonapiError` types are not used in this raml file

Reply:
I left them just to make sure that java classes generated